docs/data_collection.rst

```diff
@@ -121,49 +132,14 @@ An impression is whenever a user is shown a specific snippet.
 Snippet Clicks
 ~~~~~~~~~~~~~~
 
-Whenever a link in a snippet is clicked, we trigger an event that notes which
-particular link was clicked. This includes links that may trigger an action
-besides opening up a new page, such as links that trigger browser menus.
+Whenever a link in a snippet is clicked or a form is submitted, we trigger an event
+that notes which particular link was clicked. This includes links that may trigger
+an action besides opening up a new page, such as links that trigger browser menus,
+as well as buttons that submit a form.
 
-Video Plays, Pauses, Replays
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Some snippets allow users to view videos. Some of these snippets trigger events
-when the video is played or paused, when the end of the video is reached, or
-when the user replays the video after it finishes.
-
-Social Sharing
+Snippet Blocks
 ~~~~~~~~~~~~~~
 
-Some snippets contain popup windows to share content on social networks, such as
-Facebook or Twitter. Most of these snippets trigger an event when the user
-launches the popup window.
-
-Default Browser
-~~~~~~~~~~~~~~~
-
-Some snippets trigger an event that tracks whether Firefox is the default
-browser on the user's system. These snippets also trigger an event when the user
-makes Firefox their default browser by either clicking a link in the snippet or
-by setting the default outside of the browser.
-
-Browser UI Events
```

Comment:
This is covered in the Clicks section so I thought it wouldn't be necessary, does that sound right?

Reply:
Yep, makes sense.